src/components/Grid/Grid.styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import { deviceSizes } from "../../global/device-sizes";
+
+export const GridLayout = styled.section`
+    width: ${(props) => props.maxColumnCount < 4 ? 70 : 60}%;
```

Comment:
You can destructure props in styled components: width: ${({ maxColumnCount }) => maxColumnCount < 4 ? 70 : 60}%

Reply:
Fixed everywhere